setup.py

```diff
@@ -21,7 +21,11 @@
             "pytest==2.9.1",
         ]
     },
-    dependency_links=[],
+    entry_points={
```

Comment:
i don't feel very strongly but i like `model_base` better, since this needs to be the base-class for all models.

on a related note i think it's confusing that the docstring re-uses the name `Base` for both the parameter to, and result from the call to `declarative_base`